src/common/utils/views.py

```diff
@@ -1,14 +1,16 @@
 import json
 
 from django.http import HttpResponse
+from django.apps import apps
+from django.db.models.fields.reverse_related import ManyToOneRel
+
+from .constants import get_html_input_type, NOT_REQUIRED_FIELDS_IN_FORM
```

Comment:
Let's call the whole module for imports. Ex: `common.utils.constrants import get_html_input_type`